src/effects/effectchainslot.cpp

```diff
@@ -210,6 +211,7 @@ void EffectChainSlot::setDescription(const QString& description) {
 
 void EffectChainSlot::setMix(const double& dMix) {
     m_pControlChainMix->set(dMix);
+    sendParameterUpdate();
```

Comment:
Haven't we already connected the sendParamterUpdate() function to the value change of m_pControlChainMix?

Reply:
I have tested it by changing the "mix" of some group in the developer tools, and the slot is being triggered on value change.

Reply:
Yes, but the `valueChanged` signal is not emitted when setting the ControlObject from within the class it is a member of. I have stumbled over this a few times working on Mixxx and I am not sure why it works this way, but the workaround is pretty simple.

Reply:
When we get rid of the EffectRack layer, I think we should get rid of the `mix` knob and all other unnecessary ControlObjects for the EQs and QuickEffects. We can just send the engine an update saying the mix parameter is 1 but never allow the user to change it.